### PR TITLE
Add English usecase for AIML INSTITUTE

### PR DESCRIPTION
**Pull request by RainbowMango**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR added the translated uses case according to https://github.com/karmada-io/website/pull/132.

This doc is translated by @Tingtal. 

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by RainbowMango** [comment]
@pidb @bumblebeeK could you please review this doc?
You can get the preview at: https://deploy-preview-150--karmada.netlify.app/docs/next/casestudies/ci123

**Comment by pidb** [comment]
Okay, Thanks translation it.

File: docs/casestudies/ci123.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
+# Karmada in AIML INSTITUTE
```
Comment (Poor12) [inline docs/casestudies/ci123.md:1]: Use front matter here is better.
```
---
title: xxx
---
```

File: docs/casestudies/ci123.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+![](../resources/adoptions-ci123-aries.png)
+
+
+1. The inspection system delivers CRDs to member clusters through Karmada propagation.
+2. The surface controllers of member clusters listen to CRD creation, render a pipeline based on the definition, and calculate DAGs. The controllers execute the DAGs and generate the inspection results.
+3. The control plane runs a detector to collect, aggregate, and calculate the inspection results. Perceptible link states of member clusters allow more effective cross-cluster deployment and scheduling. The following figure shows the Cluster Inspection page in the online system.
+   ![Cluster inspection](../resources/adoptions-ci123-cluster-inspection.png "Cluster inspection")
```
Comment (Poor12) [inline docs/casestudies/ci123.md:135]: Add a blank line between here，so the picture and the word are not crowded in a row.

File: docs/casestudies/ci123.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+4. The inspection system also draws the time sequence states of the inspected links at different time points for tracing exceptions.
+
+![Time sequence state](../resources/adoptions-ci123-sequence-status.png "Time sequence state")
+
+
+## Integration of Karmada and Community Ecosystem
+
+Karmada's design makes it easy to integrate with other community projects. The following figure shows the architecture of integrating Karmada with Velero for multi-cluster backup and restore. The multi-cluster backup and restore CRDs of Velero are added to the Karmada control plane, and propagated to specified member clusters according to the Karmada propagation policies. The Backup and Restore controllers reconcile the states and aggregate the CRD states of Velero in the member clusters.
+![Time sequence state](../resources/adoptions-ci123-velero.png "Time sequence state")
```
Comment (Poor12) [inline docs/casestudies/ci123.md:145]: ditto

File: docs/casestudies/ci123.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+
+
+## Karmada for Multi-cluster Heterogeneous Resources
+
+The following figure shows how to manage multi-cluster GPU resources based on Karmada.
+![GPU resource management](../resources/adoptions-ci123-gpu-resources.png "GPU resource management")
```
Comment (Poor12) [inline docs/casestudies/ci123.md:151]: ditto

**Comment by bumblebeeK** [comment]
/ltgm
Appreciate that！

**Comment by RainbowMango** [comment]
Thanks for your confirmation.

**Pull request merged**